sdk/personalizer/azure-ai-personalizer/README.md

```diff
@@ -0,0 +1,9 @@
+# Azure Personalizer client library for .NET
+
+[Azure Personalizer](https://docs.microsoft.com/azure/cognitive-services/personalizer/)
+is a cloud-based service that helps your applications choose the best content item to show your users. You can use the Personalizer service to determine what product to suggest to shoppers or to figure out the optimal position for an advertisement. After the content is shown to the user, your application monitors the user's reaction and reports a reward score back to the Personalizer service. This ensures continuous improvement of the machine learning model, and Personalizer's ability to select the best content item based on the contextual information it receives.
+
+## Getting started
+
+### Install the package
+TBD
```

Comment:
We should add a follow-up issue to track documentation updates for the package.

Reply:
Will do. I added a task in my sprint for our product backlog task.